src/tools/emacs.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Emacs
+description: Use Dart on Emacs and its flavors
+toc: false
+---
+
+Emacs has a few major and minor modes built for Dart and Flutter development.
+
+All the steps are identical for enabling Flutter since
+[Dart-Snippets][dart-snippets] contain Flutter snippets too.
```

Comment:
At this point the concept of "snippets" hasn't been introduced, so this sentence really means very little out of context (and there is no context so far). I don't know what it means, and I've used Emacs since the 1990s and was one of Flutter's founding members. :-)

Reply:
True, I just assumed that the term "Snippet" is universally accepted and defined. It was like defining modes in Emacs or compiler!

That's why I thought it's trivial...

Thanks for your kind review. I'll add more commits toward this. Although I still believe that this term doesn't need much more explanations.